### PR TITLE
Feature/dockerize api

### PR DESCRIPTION
**Pull request by socunanena**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+    depends_on:
+      - db
 
   db:
     image: mongo
     container_name: db
+    ports:
+      - 27017:27017
     volumes:
+      - ./api/data/db:/data/db
```
Comment (socunanena) [inline docker-compose.yml]: I'm not sure where is the best location for the app data persistence... I was doubting between this one and `/data/db`. What do you think?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,13 +1,19 @@
 version: "3"
 
 services:
-  list-app:
-    build: ./list-app
-
   api:
     build: ./api
+    image: prison-wods-api
```
Comment (socunanena) [inline docker-compose.yml:6]: This way we can name the resulting image (**prison-wods-api**). If we don't provide this entry the name is autogenerated like this: "projectName_service", in our case, "**prison-wods_api**". Which one do you prefer?

File: api/.dockerignore [hunk 3]
```diff
@@ -4,3 +4,7 @@ yarn-error.log
 
 # Docker
 .dockerignore
+Dockerfile
+
+# Other
+README.md
```
Comment (pachoning) [inline api/.dockerignore:10]: I think this should be included in the image.

File: docker-compose.yml (hunk 2 above)
Comment (pachoning) [reply docker-compose.yml:6]: I prefer prison-wods-api

**Review by pachoning — APPROVED** [review]

**Pull request merged**